### PR TITLE
Fix Playback Mode Sample Rate

### PR DESCRIPTION
**Pull request by retiutut**

### CODE REVIEW

File: OpenBCI_GUI/DataSourcePlayback.pde [hunk 1]
```diff
@@ -55,7 +55,7 @@ class DataSourcePlayback implements DataSource, AccelerometerCapableBoard, Analo
             // some boards have configurable sample rate, so read it from header
             if (line.startsWith("%Sample Rate")) {
                 int startIndex = line.indexOf('=') + 2;
-                int endIndex = line.indexOf("Hz") - 1;
```
Comment (retiutut) [inline OpenBCI_GUI/DataSourcePlayback.pde:58]: Here is the change.

**Comment by retiutut** [comment]
I also see that Travis build is failing? Doesn't seem like it is caused by this PR.

**Comment by daniellasry** [comment]
> I also see that Travis build is failing? Doesn't seem like it is caused by this PR.

Could you go into travis.yml and replace all instances of `https` with `http`? That should fix it!

**Comment by retiutut** [comment]
Thanks, will try next.

File: OpenBCI_GUI/DataSourcePlayback.pde (hunk 1 above)
Comment (daniellasry) [reply OpenBCI_GUI/DataSourcePlayback.pde:58]: What was the bug? Was it failing to parse properly because if the space before `Hz`?

Comment (retiutut) [reply OpenBCI_GUI/DataSourcePlayback.pde:58]: Yep. Added a println and the sample rate was 25 instead of 250.

Comment (daniellasry) [reply OpenBCI_GUI/DataSourcePlayback.pde:58]: That's weird. If you decrease the end index it shoudl make it worse, not better. Are you sure this fixes the problem?

Comment (retiutut) [reply OpenBCI_GUI/DataSourcePlayback.pde:58]: Yes, this increases the end index by removing ` - 1`.

Comment (retiutut) [reply OpenBCI_GUI/DataSourcePlayback.pde:58]: Hold on there's more going on here...

**Pull request closed without merging**